Source/WebCore/platform/network/cf/ResourceRequestCFNet.h

```diff
@@ -120,8 +120,9 @@ inline RetainPtr<CFStringRef> httpHeaderValueUsingSuitableEncoding(HTTPHeaderMap
 {
     if (header.keyAsHTTPHeaderName && *header.keyAsHTTPHeaderName == HTTPHeaderName::LastEventID && !header.value.containsOnlyASCII()) {
         auto utf8Value = header.value.utf8();
+        auto utf8ValueSpan = utf8Value.span();
         // Constructing a string with the UTF-8 bytes but claiming that it’s Latin-1 is the way to get CFNetwork to put those UTF-8 bytes on the wire.
```

Comment:
This seems like a super strange workaround for an issue in CFNetwork. I wonder if this can ever be rectified.